docker-compose.dind.yml

```
@@ -0,0 +1,25 @@
+# Docker Compose override file for Docker-in-Docker (DIND) mode
+# This file adds:
+# - Predictable container names for DIND container management  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)